src/css/styles.scss

```diff
@@ -609,7 +614,7 @@
   }
 
   .App-toolbar--mobile {
-    overflow-x: auto;
+    overflow: visible;
```

Comment:
this will make it not scrollable on small mobile screens

![image](https://github.com/excalidraw/excalidraw/assets/5153846/8dea4e66-b738-4f26-93b2-258c8941ef96)

Reply:
Exactly why do we want it to be scrollable? Hence I have changed the CSS a bit as well so it works on most of the mobile screens. 
We shouldn't anyways have horizontal scroll for the below reasons
1. Its a bad UX and not intuitive at all
2. We can just move one of the tools inside "more tools" or on the side to support it in more small small screens up to 320px which is given by A11y guidelines.

Right now with this change I am hoping it would work well in most of the screens but as suggested I would like to move 1-2 tools inside more so UX is less cluttered and that we can do for smaller screens only after certain threshold and not for mobile layout

Reply:
Before we do a mobile redesign the menu at least has to be _usable_ on small screens. Bad UX is definitely better than "not possible to select a tool at all".

Reply:
Well, what you are talking about is just ~45px (below 365 to 320px) what we need to support, which is very small range and I am not sure if it's a standard mobile screen, but in any case what issue do you see If we move one of the tools inside more tools or on the side for this 45px range?

For other sizes, the current UX is much better and intuitive

Reply:
@dwelle in case you missed the above message ^

Reply:
let's do this and hope that there aren't many screens smaller than that.

> in any case what issue do you see If we move one of the tools inside more tools or on the side for this 45px range

We can , but I'd do it conditionally, only if the screen size won't fit the tools. And I'm not sure which tool we'd hide there (diamond?).

Reply:
I think since one of the tools will go inside the dropdown the eraser should go as that's the tool on extreme right when the width is exceeding so might be more intuitive to push the eraser instead of picking one of the shapes tools.
wdyt?

Reply:
For now, we would probably keep as is as this consideration is just for ~45px as mentioned here - https://github.com/excalidraw/excalidraw/pull/6920#discussion_r1315564739 and I am not sure if this is something we need to be worried about so we can ship it as is and later push one of the items to dropdown if this becomes a concern

Reply:
yeah, in the end we decided to leave as is as it affects only a very small mobile screen, and long-term we plan to redesign mobile completely.

Reply:
Alright as I was fixing the regression of mobile toolbar, I looked into this again and I have made it work for smaller screens up to 320px 


<img width="549" alt="Screenshot 2023-09-21 at 5 52 01 PM" src="https://github.com/excalidraw/excalidraw/assets/11256141/20445192-ecc7-448f-a92d-d25f0357e0bf">